File: Qt/view/VocHolder.cpp
#include "VocHolder.h"
#include "ui_VocHolder.h"
#include "../util/Event.h"

VocHolder::VocHolder()
: ui(new Ui::VocHolder)
{
	ui->setupUi(this);
}

VocHolder::~VocHolder()
{
	delete ui;
}

void VocHolder::fillLayout()
{
	Event* e = activeEvent();
	if (e) {
		ui->mText0->setText(e->normalizedRangeString());
		setColorByEventMode(ui->mText0, e);
	}
	else {
		ui->mText0->setText("");
	}
}

File: Astromaximum-eclipse/src/com/astromaximum/android/view/VocHolder.java
package com.astromaximum.android.view;

import com.astromaximum.android.R;
import com.astromaximum.android.util.Event;

public class VocHolder extends ViewHolder {

	public VocHolder(SummaryItem si) {
		super(si, R.layout.item_voc, LAYOUT_FLAG_TEXT0);
	}

	@Override
	public void fillLayout() {
		Event e = getActiveEvent();
		if (e != null) {
			mText0.setText(e.normalizedRangeString());
			setColorByEventMode(mText0, e);
		} else {
			mText0.setText("");
		}
	}
}
